hassio/src/resources/hassio-style.ts

```diff
@@ -26,22 +26,34 @@ export const hassioStyle = css`
   }
   .card-group {
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    grid-template-columns: repeat(
+      auto-fit,
+      minmax(var(--card-group-min-width, 300px), 1fr)
```

Comment:
This is not a good fix. It means that now we will never be able to change this grid again without knowing how it works, because it can all be variable. I think that we should just allow word wrapping the text for diagnostics by just applying our own flex styling.